lib/signal_tower/stats.ex

```diff
@@ -19,10 +18,27 @@ defmodule SignalTower.Stats do
     Process.monitor(pid)
```

Comment:
Since you're now tidying up when explicitly receiving :room_closed and not when receiving the DOWN message by the room, this line needs to be removed.